index.html

```diff
@@ -9,6 +9,23 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="book-container column">
+    <div class="row">
+        <h1 class="book-title">Título del libro</h1>
+        <p class="book-gender">(género)</p>
+    </div>
+    <div class="row">
+        <h3 class="book-author text-1">Autor del libro:</h3>
```

Comment:
Why don't we use a more generic approach here ? like setting this class name as `book-attribute` or `book-property` that way we don't have to set different names for each one.

Reply:
Done